ext/net/01_net.js

```diff
@@ -58,6 +58,10 @@
     return core.opAsync("op_dns_resolve", { query, recordType, options });
   }
 
+  function opSetNoDelay(rid, noDelay) {
+    return core.opAsync("op_set_nodelay", { rid, noDelay });
```

Comment:
I'd make this sync, it doesn't need to be async. See op_net_listen for an example of a sync op.

Reply:
Flatten args in opcall:

```suggestion
    return core.opAsync("op_set_nodelay", rid, noDelay);
```

Given that there's a single call site and with flattened args it's just a passthrough function, you could probably remove `opSetNoDelay` and just do the opcall in `Conn::setNoDelay`

Reply:
To modify the TcpStream, I need to use `self.wr_borrow_mut()` which returns `AsyncBorrowFutureImpl`, which means I have to use async/await. Is there any workaround for that? I'm quite new with AsyncRefCell in Deno. Would be nice to receive help on this, thanks!

Reply:
Changed to sync. Let me know if the implementation is correct / according to style.